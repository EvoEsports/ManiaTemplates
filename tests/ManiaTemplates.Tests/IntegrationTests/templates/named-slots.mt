<component>
    <import component="SlotsComponent" as="SlotsComponent"/>

    <property type="string" name="testVariable"/>

    <template>
        <!-- Section 1 -->
        <SlotsComponent>
            <label text="test"/>

            <template slot="footer">
                <label text="{{ testVariable }}"/>
            </template>
        </SlotsComponent>
        
        <!-- Section 2 -->
        <SlotsComponent>
            <SlotsComponent>
                <label text="test2"/>

                <template slot="footer">
                    <label text="{{ testVariable }}_footer2"/>
                </template>
            </SlotsComponent>

            <template slot="footer">
                <label text="test3"/>
                <SlotsComponent>
                    <label text="test4"/>
                </SlotsComponent>
            </template>
        </SlotsComponent>
    </template>
</component>
