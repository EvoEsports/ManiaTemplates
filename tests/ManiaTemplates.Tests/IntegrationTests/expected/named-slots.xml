<manialink version="3" id="MtNamedSlots" name="EvoSC#-MtNamedSlots">
<!--  Section 1  -->
<frame class="outer">
<label text="test" />
<frame class="inner">
<label text="UnitTest" />
</frame>
</frame>
<!--  Section 2  -->
<frame class="outer">
<frame class="outer">
<label text="test2" />
<frame class="inner">
<label text="UnitTest_footer2" />
</frame>
</frame>
<frame class="inner">
<label text="test3" />
<frame class="outer">
<label text="test4" />
<frame class="inner">
</frame>
</frame>
</frame>
</frame>
</manialink>
